book/jvm-build-service.md

```diff
@@ -163,6 +168,27 @@ spec:
   enableRebuilds: "true"
 ```
 
+The `JBSConfig` is also where shared registries are configured. A shared registry is one that may be shared by many
+users to avoid multiple rebuilds of the same artifact. This assumes the first user has configured their registry to
+deploy to a shared registry. For example:
```

Comment:
```suggestion
In order to avoid multiple rebuilds of the same artifact, a user may configure a shared registry with a `JBSConfig`.
Once a user has configured a registry, the CR can be used to point to the shared registry. For example:
```

Reply:
I assume that creating a registry (or describing the process/what it is) is out of scope for this document. Is there any other information that could be added here for clarity?

Reply:
Yes, I agree that other information on JBSConfig is missing but I am slightly loath to shoehorn that into this PR as I think its a separate piece of work.  I included the summary line
```
This assumes that another user has configured their registry to deploy builds to `my-team-owner`. For example:
```